### PR TITLE
Line Graph Component

### PR DESCRIPTION
**Pull request by richarddubay**
# Feature / Fixed Issue(s)
- Line Graph Component

# Testing/Documentation
- [x] All significant new logic is covered by tests.
- [x] Storybook documentation has been added.

# Screenshots
- If applicable, include screenshots of UI/style changes.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://9187c7a4-943f-4e0c-85ab-8e6410b53f7c.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/component-lineGraph/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by richarddubay** [comment]
@jbaxleyiii This is ready for your review.

File: imports/components/graphs/lineGraph/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// @flow
+import {
+  VictoryChart,
+  VictoryAxis,
+  VictoryLine,
+  VictoryScatter,
+} from "victory";
+
+const styles = {
```
Comment (jbaxleyiii) [inline imports/components/graphs/lineGraph/index.js]: @richarddubay I'd love to make the critical parts of this style object configurable via props:

`lineColor`
`dotColor` // for scatter
`tickStyles` // an object to override

**Comment by richarddubay** [comment]
@jbaxleyiii I put the tickFormat information in the data object, but I couldn't remove the month because Victory needs unique keys for that, otherwise it starts doubling data points for the items that have the same name. Think "April" and "August". They have separate data points, but if they are both called "A", then they get plotted as two points under the same axis point. So I just added "tick" as a third part of the object. If you're happy with this, I think we're done here.

**Comment by richarddubay** [comment]
screenshot 2016-11-30 20 49 37

**Pull request merged**